tests/test_barostat.py

```diff
@@ -157,21 +157,21 @@ def test_barostat_is_deterministic():
     seed = 2021
     np.random.seed(seed)
 
-    box_vol = 27.08816
+    box_vol = 26.89966
```

Comment:
why did this change?

Reply:
Because we are only running 15 steps rather than 1000, which means the number of times the barostat runs differs. Its also not minimized, which also has an impact. Since the test is just to check if the results are deterministic, not reason to run more steps than necessary

Reply:
Ah, got it -- it wasn't clear to me initially whether this was a parameter or expected output